docker/build_support/find-oldest-branch-root.sh

```diff
@@ -2,5 +2,11 @@
 
 set -e
 
-git log --pretty='format:%h %an' | grep GithubAction | head -n 1 | cut -d' ' -f1
+if ! [[ -z "$TARGET_ROOT_BRANCH "]]; then
```

Comment:
Rather than history manipulation, require explicit definition of the root branch based on the workflow in CI.